.gitignore

```diff
@@ -6,6 +6,7 @@
 
 # Ignore generated resource files
 Resource.designer.cs
+Resource.Designer.cs
```

Comment:
Maybe consider using the accepted answer to http://stackoverflow.com/questions/11116636/how-do-i-ignore-file-extensions-in-git-regardless-of-case

(not the config one as we can't control client side config)